packages/grid/x-data-grid-premium/src/components/GridPremiumColumnMenu.tsx

```diff
@@ -0,0 +1,65 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import {
+  GridGenericColumnMenu,
+  GridColumnMenuProps,
+  GRID_COLUMN_MENU_COMPONENTS,
+  GRID_COLUMN_MENU_COMPONENTS_PROPS,
+  GridColumnMenuItemProps,
+} from '@mui/x-data-grid-pro';
+import { GridColumnMenuAggregationItem } from './GridColumnMenuAggregationItem';
+import { isGroupingColumn } from '../hooks/features/rowGrouping';
+import { GridColumnMenuRowGroupItem } from './GridColumnMenuRowGroupItem';
+import { GridColumnMenuRowUngroupItem } from './GridColumnMenuRowUngroupItem';
+
+function GroupingItem(props: GridColumnMenuItemProps) {
+  const { colDef } = props;
+  if (isGroupingColumn(colDef.field)) {
+    return <GridColumnMenuRowGroupItem {...props} />;
+  }
+  if (colDef.groupable) {
+    return <GridColumnMenuRowUngroupItem {...props} />;
+  }
+  return null;
+}
+
+function AggregationItem(props: GridColumnMenuItemProps) {
+  return <GridColumnMenuAggregationItem {...props} />;
+}
+
+export const GRID_COLUMN_MENU_COMPONENTS_PREMIUM = {
+  ...GRID_COLUMN_MENU_COMPONENTS,
+  ColumnMenuAggregationItem: AggregationItem,
```

Comment:
```suggestion
export const GRID_COLUMN_MENU_COMPONENTS_PREMIUM = {
  ...GRID_COLUMN_MENU_COMPONENTS,
  ColumnMenuAggregationItem: GridColumnMenuAggregationItem,
```
Looks like `AggregationItem` wrapper isn't necessary here